packages/react/src/components/List/List.test.tsx

```diff
@@ -218,4 +219,79 @@ describe('List', () => {
       expect(rows).toHaveLength(100);
     });
   });
+
+  describe('utils', () => {
```

Comment:
I think these tests should go in a file name `scroll.test.ts` that lives next to `scroll.ts`.